benchmarks/block_store.py

```diff
@@ -288,10 +288,10 @@ async def run_add_block_benchmark(version: int):
             print("profiling get_full_blocks_at")
 
         start = monotonic()
-        for h in range(1, block_height):
-            blocks = await block_store.get_full_blocks_at([h])
+        for hi in range(1, block_height):
+            blocks = await block_store.get_full_blocks_at([hi])
             assert len(blocks) == 1
-            assert blocks[0].height == h
+            assert blocks[0].height == hi
```

Comment:
`blocks` is `Any` so even with strict equality checks this would allow the `int` to be compared with the ``uint32`.

Reply:
the reason I had to rename this variable is that it's also used as `bytes32` in a few places.

Reply:
Yeah, I saw that.  That's what the other comment was about, offering more descriptive names.  The comment here was just pointing out a further hinting 'issue'.  Sort of.